canvas_account_admin_tools/requirements/local.txt

```diff
@@ -20,3 +20,4 @@ pyvirtualdisplay
 # icommons_common and into ccsw, work around the circular dependency by calling
 # out the ccsw dependency here.
 git+ssh://git@github.com/Harvard-University-iCommons/django-canvas-course-site-wizard.git@v1.3.8#egg=django-canvas-course-site-wizard==1.3.8
+git+ssh://git@github.com/Harvard-University-iCommons/selenium_common.git@develop#egg=selenium-common
```

Comment:
We should figure out a versioning scheme for selenium-common, so we don't hit version mismatch issues.